ani-cli.1

```diff
@@ -0,0 +1,101 @@
+.\" Automatically generated by Pandoc 2.9.2.1
+.\"
+.TH "ANI-CLI (1) ani-cli" "" "\[lq]June 19 2022\[rq]" "" ""
```

Comment:
If we update the manpage the least that should change is this date. Possibly the authors as well.

Reply:
I am not sure what changes should be introduced to authors.